### ISSUE TITLE
x3/x4/x8/x16 Fast Forward Speed and Turbo Button 

### ISSUE THREAD

**Issue by JordaNx147** [Swordfish90/Lemuroid#837]
**Is your feature request related to a problem? Please describe.**
Would love to be able to farm exp faster in certain games. Different fast forward speeds would be great. Adding a Turbo hotkey to hold and make dialogue boxes easier.

**Describe the solution you'd like**
More options for fast forward and an extra turbo key added. Something similar to myBoy!

**Describe alternatives you've considered**

**Additional context**
Love the emulator! A couple features away from perfection.

**Comment by s0mt0chukwu** [comment]
Not an essential feature

**Comment by HFPasfho** [comment]
> Not an essential feature

I strongly disagree. Old RPGs or even pokemon games are literally UNPLAYABLE without fast forward.

**Comment by HFPasfho** [comment]
@s0mt0chukwu also you're not even a contributor. On what basis do you feel entitled to lecture @JordaNx147 what is a necessary function and what is not? LMAO

**Comment by Gamer750** [comment]
I would also like to see this feature

**Comment by Flameplayercpu** [comment]
> I would also like to see this feature
Me too

**Comment by NobQuaffer** [comment]
Me too! Either to add an onscreen turbo button or map it to a physical button would be awesome

**Comment by Pokemetti** [comment]
I just wanna comment that the 2x speed is actually hardcoded inside this project, and higher speeds are definitely possible. I hardcoded it to 8x for Pokémon and it worked fine, though mGBA uses a lot of CPU for that, since it doesn't have any dynarec/JIT recompilation as far as I know.
I sadly have no idea how to implement this as an option in the frontend, I'm just your backend guy.
